### PR TITLE
Update help window by displaying additional information

### PR DESCRIPTION
**Pull request by DannyDakota**
Inclusion of format and example for commands. New PR from #37 with updated test cases to pass CI

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -15,12 +15,71 @@
...
+            + "\n4. EDIT the person at the specified INDEX. The index refers to the index number shown in the "
+            + "displayed person list."
+            + NEWLINE_AND_INDENTATION
+            + "Format: edit INDEX n/NAME ..."
+            + NEWLINE_AND_INDENTATION
+            + "Example 1: edit 1 p/91234567 e/johndoe@example.com"
+            + NEWLINE_AND_INDENTATION
+            + "Example 2: edit 2 n/David Limpeh t/\n"
+
+            + "\n5. CLEAR all the data inside Tracey"
```
Comment (cwq2326) [inline src/main/java/seedu/address/ui/HelpWindow.java:58]: Can inform the user that the deleted data cannot be retrieved

**Review by cwq2326 — COMMENTED** [review]
Just some minor suggestions for the add and clear command. otherwise LGTM 😸

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,12 +15,71 @@
...
+            + "For a full comprehensive guide, please refer to " + USERGUIDE_URL
+            + "\n"
+
+            + "\n1. FIND a particular contact in Tracey."
+            + NEWLINE_AND_INDENTATION
+            + "Format: find NAME"
+            + NEWLINE_AND_INDENTATION
+            + "Example 1: find John\n"
+
+            + "\n2. ADD a student with relevant details into Tracey."
```
[the remark this answers is not in the document]
Comment (DannyDakota) [reply src/main/java/seedu/address/ui/HelpWindow.java:32]: Noted! I think this is good, I've updated the code accordingly 😄

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (DannyDakota) [reply src/main/java/seedu/address/ui/HelpWindow.java:60]: Great! I think this is something important to add in. I've added it accordingly into the code! 👍

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -15,12 +15,75 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s2-cs2103t-t12-3.github.io/tp/UserGuide.html";
+    public static final String NEWLINE_AND_INDENTATION = "\n     ";
+    public static final String HELP_MESSAGE = "Tracey supports the following features: Find, Add, Delete, Edit, Clear, "
```
Comment (jiale-c) [inline src/main/java/seedu/address/ui/HelpWindow.java:20]: I feel that this looks organised! :relaxed:

**Review by jiale-c — APPROVED** [review]
LGTM! Nice!

**Pull request merged**